source/of.js

```diff
@@ -1,9 +1,10 @@
-import _curry1 from './internal/_curry1.js';
-import _of from './internal/_of.js';
-
+import _curry2 from './internal/_curry2.js';
 
 /**
- * Returns a singleton array containing the value provided.
+ * Given a constructor and a value, returns a new instance of that constructor
+ * containing the value.
+ *
+ * Dispatches to the `of` method of the first argument, if present.
```

Comment:
Should we mention `fantasy-land/of`?

Reply:
Probably a good idea :) Will change.

Reply:
Done. Not too sure about the description now. Let me know if you have some other suggestions.